examples/v1beta1/nodeclaim/manual-nodeclaim.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: karpenter.sh/v1beta1
+kind: NodeClaim
+metadata:
+  name: manual-kaito
+  labels:
+    karpenter.sh/nodepool: kaito
+  annotations:
+    karpenter.sh/do-not-disrupt: "true"
```

Comment:
I wonder if the [forceful Expiration RFC](https://github.com/kubernetes-sigs/karpenter/pull/1303) Upstream messes with kaito since we rely on do-not-disrupt to prevent these nodes manually scaled up from ever being removed and we don't actually have a real nodepool. 

I need to learn more about nodeclaim lifecycle in general to see how it works in this case where we have a nodeclaim but no nodepool or nodeclass

Reply:
As far as I know, we have not tested this. cc @helayoty -- do kaito nodeclaims have any expiration associated with them?

@Bryce-Soghigian are you saying this would apply to any nodeclaim or would nodeclaims w/ corresponding nodes provisioned be excepted from that default?

I'll need to look at that RFC, as well.

Reply:
Yes, it may be worth checking the expiration behavior of standalone NodeClaims (irrespective of Forceful Expiration, which would still respects do-not-disrupt). I strongly suspect that expiration just won't work, since all configuration for it (including defaults) comes from NodePool, but worth checking nonetheless.